README.md

```diff
@@ -1,10 +1,10 @@
 # Canopy
 
 <p align="center">
-<a href="https://pypi.org/project/fastapi" target="_blank">
+<a href="https://pypi.org/project/canopy-sdk" target="_blank">
```

Comment:
@izellevy this change is actually not related to this PR. Can you please remove it, and @miararoy will PR this straight to `main`?

Reply:
Sure